docs/how-to-start.en-US.md

```diff
@@ -0,0 +1,153 @@
+---
+order: 1
+title:
+  en-US: How to start
+  zh-CN: 如何开始
+type: Dev
+---
+
+## First, pre-order preparation
+
+As you begin your business development on `ng-alain`, I recommend that you first review the following documents, which is very helpful for everyone on the team.
+
++ [Ant Design Guide article](//ant.design/docs/spec/introduce)
++ [Ant Design for Angular](//ng.ant.design/)
++ [Antv chart](//antv.alipay.com/zh-cn/index.html)
++ [NG-ZORRO Community recommendation](https://ng.ant.design/docs/recommendation/en)
+
+`ng-alain`'s base component library comes from `ng-zorro-antd`, so you can get a very detailed [API](//ng.ant.design/) documentation for its use on the official website. For ng components provided by -alain are obtained through the [component page](/components).
+
+## Second, the startup process
+
+`ng-alain` is a scaffold that can be used directly in production environments. The prerequisite for understanding these details is that you have a certain knowledge of Angular. The following documents may be helpful to you before you start:
+
+- `ng-alain` Getting started video ([YouTube](https://www.youtube.com/watch?v=lPnNKPuULVw&list=PLhWkvn5F8uyJRimbVZ944unzRrHeujngw)、[Tencent video](http://v.qq.com/vplus/2c1dd5c6db4feeeea25e9827b38c171e/foldervideos/870001501oy1ijf)、[Station B](https://space.bilibili.com/12207877/#/channel/detail?cid=50229)）
```

Comment:
I recommend keeping the video and marking this as a Chinese version.

Reply:
It will be better to keep English content for en locale of ReadMe.md document.